src/presetsController.ts

```diff
@@ -343,28 +342,22 @@ export class PresetsController {
                         // No selection was made
                         return false;
                     } else {
-                        if (chosen_kit.kit.name === SpecialKits.ScanForKits) {
```

Comment:
If we don't reuse this cache anymore, consider remove the "scan for compiler" command (which is an alias of "scan for kits" so removing it won't cause any lost of functions). If you would like to keep it, then I would suggest to hide it in presets mode like "scan for kits".

Reply:
Other changes don't seem to be consistent with this one...

Reply:
Can you elaborate on what isn't consistent? I'm not very familiar with what this code is for. I just removed the `if` block here because all it appeared to do was `preset.setCompilers`, which is no longer needed (since I removed `kits` from preset.ts).

Reply:
Thanks for the explanation, I'll revert the changes (sans removing `preset.setCompilers`)

Reply:
As Bob said, this is an option used to generate a preset using a selected compiler. By inconsistent, I mean:
- If you have a better way to look for compilers, yes you don't need this if statement, but you'll need to change the way a compiler is selected (chosen_kit) as well.
- If you still want to use kit for compiler selection, I can't see a reason why this if statement needs to be removed.

Reply:
Gotcha, thanks for the extra info. Generating a preset from a kit still seems useful so I'll keep the if statement.